server/src/entities/face-search.entity.ts

```diff
@@ -0,0 +1,21 @@
+import { AssetFaceEntity } from 'src/entities/asset-face.entity';
+import { asVector } from 'src/utils/database';
+import { Column, Entity, Index, JoinColumn, OneToOne, PrimaryColumn } from 'typeorm';
+
+@Entity('face_search', { synchronize: false })
```

Comment:
Can we synchronize this table?

Reply:
I assumed it would try to change the column type, but I can try it.

Reply:
Yeah, it tries to change it to `real array`.